run/jobs/pom.xml

```diff
@@ -46,6 +46,12 @@ limitations under the License.
       <version>1.1.3</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-logging</artifactId>
+      <version>3.6.2</version>
+      <scope>test</scope>
```

Comment:
Why is this just `test` scope?

Reply:
The tests are looking for the logs in Cloud Logging. The sample writes to stdout.